src/NuGet.Core/NuGet.Protocol/ResourceProvider.cs

```diff
@@ -89,7 +89,7 @@ public virtual Type ResourceType
 
         private static IEnumerable<string> ToArray(string s)
         {
-            if (!String.IsNullOrEmpty(s))
+            if (!string.IsNullOrEmpty(s))
```

Comment:
Revert file

Reply:
This is to fix a compile error (as is the other).